test/common/util.ts

```diff
@@ -51,7 +51,9 @@ export function $x(
  */
 export var selectors = {
     nav: "#stage > #nav",
-    modal: "#stage",
+    components: "#stage",
     tab: "//hx-tablist/hx-tab",
-    visreg: "*[data-visreg]"
+    visreg: "*[data-visreg]",
+    singleLineText: "div.demo",
+    multiLineText: "#multi-line-text-demo"
```

Comment:
These are not a good candidates for a reusable selectors. Just inline these selectors into their respective `snap` calls directly.

```js
await snappit.snap("{browserName}/text-inputs/single-line-text", $("div.demo"));
```

and

```js
await snappit.snap("{browserName}/text-inputs/multi-line-text", $("#multi-line-text-demo"));
```